subcommands/devices/list.go

```diff
@@ -68,25 +68,27 @@ func ownerFormatter(d *client.Device) string {
 }
 
 var Columns = map[string]column{
-	"name":           {func(d *client.Device) string { return d.Name }},
-	"uuid":           {func(d *client.Device) string { return d.Uuid }},
-	"factory":        {func(d *client.Device) string { return d.Factory }},
-	"owner":          {ownerFormatter},
-	"device-group":   {func(d *client.Device) string { return d.GroupName }},
-	"target":         {func(d *client.Device) string { return d.TargetName }},
-	"status":         {statusFormatter},
-	"apps":           {func(d *client.Device) string { return strings.Join(d.DockerApps, ",") }},
-	"up-to-date":     {func(d *client.Device) string { return fmt.Sprintf("%v", d.UpToDate) }},
-	"tag":            {func(d *client.Device) string { return d.Tag }},
-	"created-at":     {func(d *client.Device) string { return d.ChangeMeta.CreatedAt }},
-	"created-by":     {func(d *client.Device) string { return d.ChangeMeta.CreatedBy }},
-	"updated-at":     {func(d *client.Device) string { return d.ChangeMeta.UpdatedAt }},
-	"updated-by":     {func(d *client.Device) string { return d.ChangeMeta.UpdatedBy }},
-	"last-seen":      {func(d *client.Device) string { return d.LastSeen }},
-	"ostree-hash":    {func(d *client.Device) string { return d.OstreeHash }},
-	"current-update": {func(d *client.Device) string { return d.CurrentUpdate }},
-	"is-prod":        {func(d *client.Device) string { return fmt.Sprintf("%v", d.IsProd) }},
-	"is-wave":        {func(d *client.Device) string { return fmt.Sprintf("%v", d.IsWave) }},
+	"name":         {func(d *client.Device) string { return d.Name }},
+	"uuid":         {func(d *client.Device) string { return d.Uuid }},
+	"factory":      {func(d *client.Device) string { return d.Factory }},
+	"owner":        {ownerFormatter},
+	"device-group": {func(d *client.Device) string { return d.GroupName }},
+	"target":       {func(d *client.Device) string { return d.TargetName }},
+	"status":       {statusFormatter},
+	"apps":         {func(d *client.Device) string { return strings.Join(d.DockerApps, ",") }},
+	"up-to-date":   {func(d *client.Device) string { return fmt.Sprintf("%v", d.UpToDate) }},
+	"tag":          {func(d *client.Device) string { return d.Tag }},
+	"created-at":   {func(d *client.Device) string { return d.ChangeMeta.CreatedAt }},
+	"created-by":   {func(d *client.Device) string { return d.ChangeMeta.CreatedBy }},
+	"updated-at":   {func(d *client.Device) string { return d.ChangeMeta.UpdatedAt }},
+	"updated-by":   {func(d *client.Device) string { return d.ChangeMeta.UpdatedBy }},
+	"last-seen":    {func(d *client.Device) string { return d.LastSeen }},
+	"ostree-hash":  {func(d *client.Device) string { return d.OstreeHash }},
+	// TODO: Check if we can fix
+	//nolint:misspell
+	"curent-update": {func(d *client.Device) string { return d.CurrentUpdate }},
```

Comment:
This line is being change in 3 commits, which creates too much friction.

@camilamacedo86 Over time you'll notice that during our PRs we do a lot of commit reorderings using e.g. `git rebase -i`.
The rationale is: we are an OS company following the _stable code_ best practices.
On practice this means that before being merged each our PR contains a logical set of commits, and each of those commits is minimal.

This might look unusual for newcomers, but it helps us produce the stable software releases of good quality.

That said, I _believe_ this is a bug, not at all linter related, which definitely needs to be fixed.
Thanks for finding this!
I'd appreciate if you could make a separate PR which fixes it.